.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+If the provided content is part of the present PR remove the # symbol.
+
+- [ ] **changelog** provided | not required because: 
+- [ ] **development PR** provided # | not required because: 
+- [ ] **framework PR** provided Mbed-TLS/mbedtls-framework# | not required
+- [ ] **3.6 PR** provided # | not required because: 
+- [ ] **2.28 PR** provided # | not required because: 
+- **tests**  provided | not required because: 
```

Comment:
How come there is no checkbox by `tests`?

Reply:
There's a check box next to PR links because often that PR won't be present from the start. There's a check box next to changelog because we often forget about it so we want an affirmative step about the changelog. Neither consideration applies to tests.